stock_putaway_product_template/models/product_product.py

```diff
@@ -0,0 +1,14 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+# Copyright 2020 Víctor Martínez - Tecnativa
+
+from odoo import fields, models
+
+
+class ProductProduct(models.Model):
+    _inherit = "product.product"
+
+    product_product_putaway_ids = fields.One2many(
```

Comment:
Is needed this field por product.product model?

Reply:
These fields it's not necessary now (similar in product_template model).
Not necessary now demo data and simplify tests.